config/base.js

```diff
@@ -97,7 +112,8 @@ module.exports = {
     cache: SCHEMA_CACHE,
     cachePerm: SCHEMA_CACHE_PERMUTATION,
     childPipelines: SCHEMA_CHILD_PIPELINES,
-    config: SCHEMA_CONFIG,
+    configBeforeMergingTemplate: SCHEMA_CONFIG_PRE_TEMPLATE_MERGE,
```

Comment:
This is a breaking change. Please follow https://www.npmjs.com/package/semantic-release to bump up `major` version